go-crud/books/controllers.go

```diff
@@ -0,0 +1,257 @@
+package books
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var data []book
+
+func loadBookFile() error {
+	_, err := os.Stat("books.json")
+
+	if os.IsNotExist(err) {
+		str, err := json.Marshal(data)
+
```

Comment:
nitpicking: These empty lines between actual expression and error handling is unnecessary.